cfme/tests/services/test_myservice.py

```diff
@@ -45,7 +46,9 @@ def myservice(setup_provider, provider, catalog_item, request):
     Metadata:
         test_flag: provision
     """
-    vm_name = catalog_item.provisioning_data["vm_name"] + "_0001"
+    vm_name = version.pick({
```

Comment:
would it make sense to standardize on minimizing the picked sub-expression? (i.e. just the vm prefix)